CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 1.2.1 (2016-04-22)
+## 1.2.1 (2016-05-03)
+
+#### Scheduler
+
+- Add containerslots filter to allow container number limit on a node
```

Comment:
Can we change this to `...to allow for limiting number of containers on a node`? It'll be more clear in my opinion.